packages/react-wrapper/package.json

```diff
@@ -37,8 +37,8 @@
   "dependencies": {
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
-    "tslib": "~1.11.1",
-    "vue": "~2.6.14"
+    "vue": "^2.6.0",
+    "tslib": "~2.3.0"
```

Comment:
Why did you reorder this? 😅

Reply:
Probably pressed the wrong keys, good catch